pennylane_qiskit/converter.py

```diff
@@ -35,17 +35,80 @@
 dagger_map = {"SdgGate": qml.S, "TdgGate": qml.T, "SXdgGate": qml.SX}
 
 
-def _check_parameter_bound(param: Parameter, var_ref_map: Dict[Parameter, Any]):
+def _check_parameter_bound(param: Parameter, trainable_params: Dict[Parameter, Any]):
     """Utility function determining if a certain parameter in a QuantumCircuit has
     been bound.
 
     Args:
         param (qiskit.circuit.Parameter): the parameter to be checked
-        var_ref_map (dict[qiskit.circuit.Parameter, Any]):
+        trainable_params (dict[qiskit.circuit.Parameter, Any]):
             a dictionary mapping qiskit parameters to trainable parameter values
     """
-    if isinstance(param, Parameter) and param not in var_ref_map:
-        raise ValueError("The parameter {} was not bound correctly.".format(param))
+    if isinstance(param, Parameter) and param not in trainable_params:
+        raise ValueError(f"The parameter {param} was not bound correctly.".format(param))
+
+
+def _format_params_dict(quantum_circuit, params, *args, **kwargs):
+    """Processes the inputs for calling the quantum function and returns
+    a dictionary of the format {Parameter("name"): value} for all the parameters.
+
+    For a ``quantum_circuit`` with parameters ``[Parameter("phi"), Parameter("psi"), Parameter("theta")]``,
+    inputs can be one of the following:
+        1. the kwargs passed to when calling the qfunc, other than ``params`` and ``wires``.
+            The keys in kwargs are expected to correspond to the names of the Parameters on
+            the QuantumCircuit, i.e. (qc, None, phi=0.35, theta=0.2, psi=1.7)
+        2. the args passed when calling the qfunc, assigned to the Parameters in alphabetical
```

Comment:
What do we mean by alphabetical order? Are we sorting them?

Reply:
Unfortunately they are sorting them, and we mean alphabetical by the name of the parameter (designated when the user initialized the Qiskit parameter as `Parameter("name")`. I'll add a sentence expanding on this to make it clearer.